seonshine_frontend/src/config/axios.ts

```diff
@@ -18,10 +19,24 @@ axiosInstance.interceptors.request.use(
     return config;
   },
   (error) => {
-    if (error.response.status === 401) {
-      console.log('logout');
-    }
+    return Promise.reject(error);
   },
 );
 
+axiosInstance.interceptors.response.use(
+  (response) => {
+    return response;
+  },
+  (error) => {
+    console.log('error', error);
+    if (error.response && (error.response.status === 403 || error.response.status === 401)) {
```

Comment:
define constant for 401 and 403 status codes. We can define new file `http.ts` in constants folder for specific status codes.
Ex: `const {status} = error?.response || {}`
and check `status === UNAUTHENTICATED`